generic/tests/cfg/timedrift.cfg

```diff
@@ -1,6 +1,7 @@
 - timedrift: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu libvirt
-    rtc_drift = "slew"
+    i386, x86_64:
+        rtc_drift = "slew"
```

Comment:
Do we have this param defined for other arch some where else?

Reply:
@vivianQizhu Thanks, this param is for i386 targets only.